<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Syn_Data_Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="OG_Tab">
     <attribute name="title">
      <string>Syn_Obj_Gen</string>
     </attribute>
     <widget class="QPushButton" name="Obj_Run_B">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>238</y>
        <width>171</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Syn_Obj_Gen Run</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="Obj_ProgressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>531</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QLabel" name="label_38">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Input Path</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="OG_Image_Directory_LE">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>361</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>../OG_Image_Path/</string>
       </property>
      </widget>
      <widget class="QPushButton" name="OG_Image_Directory_B">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>OG_Image Path</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="SIG_Tab">
     <attribute name="title">
      <string>Syn_Data_Gen</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>531</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QLabel" name="label_39">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Input Path</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Bg_Directory_LE">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>361</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>../Background_Path/</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Bg_Directory_B">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Background Path</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Obj_Image_Directory_B">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Obj_Image Path</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Image_Directory_LE">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>361</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>../Obj_Image_Path/</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Config_Run_B">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>171</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Config Strat</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>531</width>
        <height>441</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QLineEdit" name="Obj_Class_Line_1">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_6">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_7">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>45</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>85</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>205</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>165</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>125</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_7">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>85</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_6">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>45</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Class_Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_41">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Class_Number</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_42">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Class_Number</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_43">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Class_Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_8">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>125</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_9">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>165</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLabel" name="Obj_Label_10">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>205</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_8">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_9">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>160</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Class_Line_10">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_44">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Obj_Size</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_45">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Bg_Size</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_46">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_47">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>250</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Min">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Max">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>250</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Bg_Min">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>290</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_48">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>290</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Bg_Max">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>290</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_49">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Obj_Num">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>340</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_50">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Obj_Num</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_51">
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>340</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Syn_Num per Bg</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Syn_Num">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>340</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_52">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>340</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Split_Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Split_Name">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>340</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="Syn_Run_B">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>390</y>
         <width>171</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Syn_Data_Gen Run</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="Syn_ProgressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>393</y>
         <width>341</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
